### ISSUE TITLE
Disable Email Notification Options

### ISSUE THREAD

**Issue by MarcosAnjos** [IQSS/dataverse#9978]
Hello, Dataverse team!

I've been working on some customizations for Dataverse and came across a need that perhaps other users might also have. Currently, according to the [documentation](https://guides.dataverse.org/en/5.12.1/admin/user-administration.html#mute-notifications) on "Letting Users Manage Notifications", we have the :AlwaysMuted and :NeverMuted options.

I'd like to specifically disable email notifications for CREATEDS, DATASETCREATED, and PUBLISHED, but I still want these notifications to appear on the user interface.

Using the command:

`curl -X PUT -d 'CREATEDS, DATASETCREATED, PUBLISHED' http://localhost:8080/api/admin/settings/:NeverMuted
`

This will disable both email sending and interface notifications. However, is there a way to only enable interface notifications by passing some flag, or by any other means?

**Comment by pdurbin** [comment]
@MarcosAnjos nope. Currently, there's no way to distinguish between email and in-app/interface notifications.

@ErykKul implemented the feature in this PR and might be interested in your feedback:

- #8530

**Comment by ErykKul** [comment]
You can disable e-mail notifications and keep in app notification only for yourself, i.e., on the individual level. As admin, you can only "gray out" some notifications for everybody, in both lists at the same time (always muted are the unchecked ones, that the user cannot enable them, and never muted are always checked and users cannot mute them by disabling them):

![image](https://github.com/IQSS/dataverse/assets/101262459/e785ce45-8e1d-4a4e-83e4-f304ae083856)

In order to allow this control by the users, you need to turn that feature on:

https://guides.dataverse.org/en/latest/admin/user-administration.html#letting-users-manage-notifications
```curl -X PUT -d 'true' http://localhost:8080/api/admin/settings/:ShowMuteOptions```

Nevertheless, it would not be very hard to extend the current implementation and allow more fine-grained control by the admins by adding an option for disabling only e-mail notifications and keeping the in-app notifications.

**Comment by MarcosAnjos** [comment]
@ErykKul Obrigado :)
